scripts/extensions/print_keys

```diff
@@ -1,3 +1,11 @@
 #!/bin/sh
 
-erl -pa {{lib_dirs}}/**/ebin -noshell -eval 'miner_keys:print_keys("{{sys_config}}")' -eval 'init:stop()'
+CONFIG=$(find $ROOTDIR/releases -name sys.config)
+LIB_DIR=$ROOTDIR/lib
+
+if relx_nodetool "ping" > /dev/null 2>&1; then
+    cat /var/data/public_keys
```

Comment:
do an exec here, so the exit code will propogate if the file doesn't exist?

Reply:
Although you can almost certainly get the information you want out of the running swarm??

Reply:
> do an exec here, so the exit code will propogate if the file doesn't exist?

`exec` will also terminate the `print_keys` script so that `exit 0` is never reached and thereby superfluous?

Reply:
No more cat now that #335 has been merged into this PR.